_health-care/_js/components/HealthCareApp.jsx

```diff
@@ -213,8 +213,30 @@ class HealthCareApp extends React.Component {
     this.setState({ applicationData: newApplicationData });
   }
 
+  resetNullValues(objectData) {
+    const context = this;
```

Comment:
eep. I don't understand the scoping of `this` to know why this code is necessary. Can we add a comment or somehow remove?